modules/light-clients/08-wasm/types/wasm.go

```diff
@@ -11,32 +11,49 @@ import (
 )
 
 // GetCodeHashes returns all the code hashes stored.
-func GetCodeHashes(ctx sdk.Context, cdc codec.BinaryCodec) CodeHashes {
+func GetCodeHashes(ctx sdk.Context, cdc codec.BinaryCodec) (CodeHashes, error) {
 	wasmStoreKey := ibcwasm.GetWasmStoreKey()
 	store := ctx.KVStore(wasmStoreKey)
 	bz := store.Get([]byte(KeyCodeHashes))
 	if len(bz) == 0 {
-		return CodeHashes{}
+		return CodeHashes{}, nil
 	}
 	var hashes CodeHashes
```

Comment:
maybe here for good measure too! 

```suggestion
	}
	
	var hashes CodeHashes
```